bigquery/data_in_bq.md

```diff
@@ -18,12 +18,12 @@ See the sample queries about [Places](/bigquery/query_places.html) and [Properti
+DC contains over 90,000 distinct variables, a.k.a. 'statistical variables', all of whom can be explored in the [statistical variable explorer tool](https://autopush.datacommons.org/tools/statvar). Not all places have observations corresponding to every one of these variables. For example the [Average Retail Price of Electricity](https://datacommons.org/tools/statvar#Quarterly_Average_RetailPrice_Electricity) is available for all States in the US but this information is not available at the City or County level like it is for the [Count of Households with no Health Insurance](https://datacommons.org/tools/statvar#Count_Household_NoHealthInsurance) . The Map and timeline tools can not only be for visualizations, but also for determining what variables are available for a given geography. To do so, visit the [Map Explorer](https://datacommons.org/tools/map), type the name of a geographic region like "United States" or "India" and select an administrative entity level (place type) from the dropdown on the right. You will notice the Statistical Variables on the left pane filtered to the relevant set only.
 
 See the [sample queries section](/bigquery/dc_to_bq_queries.html) to experiment with variables.
 
 ## Data Sources
 
-Often different sources provide data for the same variable. Sources are named and as shown in the [sample queries](/bigquery/dc_to_bq_queries.html), we can specify a source, if needed. As an example, sources for a given series can be seen for Fertility Rate in the[Statistical Variable Explorer](https://datacommons.org/tools/statvar#FertilityRate_Person_Female). Fertility Rate data in Data Commons is sourced from the World Development Indicators and from EuroStat.
+Often different sources provide data for the same variable. As an example, sources for a given series can be seen for Life Expectancy in the [Statistical Variable Explorer](https://datacommons.org/tools/statvar#LifeExpectancy_Person). Life Expectancy data in DC is sourced from the World Development Indicators and from EuroStat, and the same place can have data from both sources. You can select the data source in the Map View or Timeline visualization by clicking on the **Select Source** button below the chart. For example, in the [Map View for Life Expectancy in all countries of the world](https://datacommons.org/tools/map#%26sv%3DLifeExpectancy_Person%26pc%3D0%26denom%3DCount_Person%26src%3D3563600999%26pd%3DEarth%26ept%3DCountry).
```

Comment:
"World bank and Organisation for Economic Co-operation and Development (OECD)" ... for Life Expectancy?

Reply:
Done